students/NatalieRodriguez/Lesson03/mailroom_final.py

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env python3
+
+#Lesson 03: Mailroom
+#Natalie Rodriguez
+#3.18.2018
+
+#list of donors and the amounts donated
+donors = [['Luke Rodriguez', 12.75, 50.31, 42.59], ['Emma Burgess', 390.87, 30.00], ['Greg Cayetano', 875.60, 1120.00],
+          ['Hannah Watson', 20.58, 1120.14], ['Emily Connor', 10.00], ['Catherine Davis', 400.00],
+          ['River Tails', 63.56, 1200.00, 300.65], ['Virginia Ferdinand', 350.35, 5000.00], ['Joseph Kibson', 3498.00, 5.50],]
+
+
+def thank_you():
+    print("\nPlease enter the name of the donor you would like to thank,\nenter a new name, or enter 'list' for current donors.\nEnter '3' to return to the MAILROOM MENU.")
+    answer = input()
+
+    if answer == "list":
+        for donor_total in donors:
+            print(donor_total[0])
```

Comment:
readability tip: this should be `for donor in donors` as you are not iterating over totals but donor entry